lib/transatl.g

```diff
@@ -169,9 +169,8 @@ BindGlobal("CheckSynonyms", function(arg)
   for p in pairs do 
     Print("Checking pair ", p, "\n");
     for n in allnames do
-      pos := PositionSublist(n, p[1]);
-      if pos <> fail then
-        n2 := SubstitutionSublist(n, p[1], p[2]);
+      n2 := ReplacedString(n, p[1], p[2]);
+      if not IsIdenticalObj(n) <> IsIdenticalObj(n2) then
```

Comment:
Yes, that's what it should be, and that's also what is in the repository. This PR was incorrect in its original form, but before I had a chance to push an updated version, it was merged -- indeed prematurely. And indeed, passings tests is a necessary condition, but not sufficient, for a PR to be OK. So @alex-konovalov and me already discussed this, and agreed that future PR reviews must be done more carefully (and IMHO that's what's been happening).

Anyway, this was resolved the next day (i.e. three weeks ago).